packages/server/database/types/MeetingTeamPrompt.ts

```diff
@@ -33,7 +33,7 @@ export default class MeetingTeamPrompt extends Meeting {
   meetingType!: 'teamPrompt'
   meetingPrompt: string
   meetingSeriesId?: number
-  scheduledEndTime?: Date
+  scheduledEndTime?: Date | null | undefined
```

Comment:
Even though it's set as possibly `undefined` field, I can't update it in rethink to `undefined`. the only way I found working is to add explicit `null` here and update it to `null` when resetting this field.

Reply:
Yea, this seems to be a pattern used elsewhere, e.g. with `nextAutoGroupThreshold` in `removeReflectionFromGroup.ts`.